data/json/recipes/chem/fuel.json

```
@@ -10,7 +10,7 @@
     "time": "30 m",
     "batch_time_factors": [ 83, 5 ],
     "autolearn": true,
-    "book_learn": [ [ "textbook_chemistry", 3 ], [ "brewing_cookbook", 3 ] ],
+    "book_learn": [ [ "textbook_chemistry", 3 ], [ "brewing_cookbook", 3 ], [ "reference_cooking", 4 ] ],
```

```suggestion
    "book_learn": [ [ "textbook_chemistry", 3 ], [ "brewing_cookbook", 3 ], [ "reference_cooking", 3 ] ],
```

autolearn is at lvl 4